DATA/production/qc-workflow.sh

```diff
@@ -87,7 +87,13 @@ elif [[ -z ${QC_JSON_FROM_OUTSIDE:-} ]]; then
       fi
     fi
     [[ -z "${QC_JSON_CPV:-}" ]] && QC_JSON_CPV=consul://o2/components/qc/ANY/any/cpv-physics-qcmn-epn
-    [[ -z "${QC_JSON_TRD:-}" ]] && QC_JSON_TRD=consul://o2/components/qc/ANY/any/trd-full-qcmn-nopulseheight-epn
+    if [[ -z "${QC_JSON_TRD:-}" ]]; then
+      if has_detectors_reco ITS TPC TRD && has_detector_matching ITSTPCTRD
```

Comment:
```suggestion
      if has_detectors_reco ITS TPC TRD && has_detector_matching ITSTPCTRD; then
```
Also in the three other blocks

Reply:
thats a complete fail of copy and paste.